app/posts/personalised-prevention-platform/2025/03/2025-03-31-technical-research.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Technology research summary"
+description: A summary of technology within NHSE that could support personalised prevention work
+date: 2025-03-31
+tags:
+- discovery
+---
+
+As part of the Personalised Prevention Platform discovery, we examined the existing and planned NHS England technology platforms and products so we could understand what could be reused or extended, and what we might need to build from scratch.
+
+## Problem statement
+
+We need to understand the availability and readiness of the NHS platforms and data we could use to identify and select people with certain characteristics and risk factors.
+
+Our current problem statement will continue to evolve over the course of discovery and alpha as we learn more about the problem we are trying to understand, and the hypotheses we are aiming to prove or disprove.
+
+## Purpose
+
+This analysis aims to give a simplified view of what NHS services, platforms and data could be available to technical solutions in the personalised prevention space.
+
+We explored 4 overarching hypotheses in the Personalised Prevention Platform Discovery  However, for the purposes of technology research, we focused on these 2 hypotheses:
+1. There is a serviceable cohort of people who will proactively manage their health to reduce their need for future treatment.
+2. We will be able to reach people early enough before health issues become problematic.
+
+This analysis is aimed at Product and Service colleagues in NHS England so they can visualise and understand how the available technology and data can best be used to feed into design considerations.
+
+## Background
+
+In the context of this early‑stage technology research, personalised prevention can be prompted by proactive campaigns for cohorts of people with shared characteristics, clinical conditions and risk factors.
+
+## What we explored
+
+### Understanding more about how to create people cohorts with shared characteristics and risk factors
+
+We wanted to understand what NHS England:
+- technology we can use to create, manage and enable the cohorting of people with shared characteristics
+- data sets and collections we can use to create cohorts of people
+- technology we can use to communicate with cohorts of people
+
+## What we did
+
+### Research the relevant NHS technology and data landscape
```

Comment:
Can probably remove the H4 level by collapsing some of the h2/3s into single headers - but again, just a comment, not a _requirement_ :)